### ISSUE TITLE
Setup atlantis for github

### ISSUE THREAD

**Issue by 6za** [kubefirst/kubefirst#217]
```bash 
kubectl -n atlantis logs atlantis-0


Error: --gh-user/--gh-token or --gh-app-id/--gh-app-key-file or --gh-app-id/--gh-app-key or --gitlab-user/--gitlab-token or --bitbucket-user/--bitbucket-token or --azuredevops-user/--azuredevops-token must be set
```

**Comment by 6za** [comment]
``` 
atlantis                    atlantis-0                                                   0/1     CrashLoopBackOff   5          4m10s
```

**Comment by 6za** [comment]
Atlantis, seems to need some extra config to run on github, it may need to wait github runner PRs from @pagottoo

**Comment by 6za** [comment]
![image](https://user-images.githubusercontent.com/53096417/184220703-faf42a1f-7ef6-4b88-815e-6b332de0e79c.png)

**Comment by 6za** [comment]
#223 Addressed on boarding of 1.9.x candidate.
